src/pyosrd/agents/scheduler_agent.py

```diff
@@ -254,13 +259,13 @@ def regulate_scenarii_with_agents(
     elif scenarii in OSRD.scenarii:
         scenarii = [scenarii]
     elif isinstance(scenarii, str):
```

Comment:
Not really sure of this part. What happens if in the list of scenarii, the first ones are ok but one following is mistyped, eg: `['c1_delay', 'foo']` ? Does it compute the first ones and raise an error afterwards (hence losing all) ? Or does it check that all are ok before running anything ?

Reply:
The check occurs in regulate_scenario therefore it will compute the first ones before raising an error.

Not sure if it would be best to check every scenario before.

Reply:
TODO Check all before starting the run